crates/router/src/routes/app.rs

```diff
@@ -685,6 +685,10 @@ impl Forex {
                 web::resource("/convert_from_minor").route(web::get().to(currency::convert_forex)),
             )
     }
+    #[cfg(feature = "v2")]
+    pub fn server(state: AppState) -> Scope {
+        web::scope("/forex").app_data(web::Data::new(state.clone()))
+    }
```

Comment:
This is an unreachable code, we can remove this here and other occurences as well